src/controllers/commands/CommandModel.hpp

```diff
@@ -11,8 +11,16 @@ class CommandController;
 
 class CommandModel : public SignalVectorModel<Command>
 {
+public:
     explicit CommandModel(QObject *parent);
 
+    enum Column {
+        Trigger = 0,
+        CommandFunc = 1,
+        ShowInMessageContextMenu = 2,
+        COUNT
```

Comment:
Makes potential future diffs cleaner. Always append extra comma after the last element in QList / enum literals like this
```suggestion
        COUNT,
```